portfolio/src/main/webapp/script.js

```diff
@@ -64,35 +63,43 @@ function initMap() {
   const griffithMarker = new google.maps.Marker({
     position: griffith,
     map: map,
-    icon: blueIcon});
+    icon: blueIcon,
+    title: "Griffith Observatory"});
```

Comment:
Same here, adding trailing commas (at the end of "Griffith Observatory") so that the ending braces go on a new line can help keep things neat and make it easy if we want to add more properties to the marker.